### ISSUE TITLE
[Adaptive Cards -Publish Adaptive Card Schema]: Ensures <img> elements have alternate text or a role of none or presentation.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8538]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Version 23H2 (OS Build 25381.1)
URL: https://adaptivecards.io/designer/
Browser: Edge Dev Version 115.0.1880.3 (Official build) dev (64-bit)
Tools: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to 'Designer' tab item and press enter key to activate it.
3. Run accessibility insight for web tool and observe the issue. 

**Actual Result:**
<img> elements does not have alternate text or a role of none or presentation as screen reader is announcing only as **'Group'.**

**Expected Result:**
Ensures <img> elements should have alternate text or a role of none or presentation.

**Element path:**
.ac-image

**Snippet:**
<img class="ac-image" src="[https://pbs.twimg.com/profile_images/3647943215/d7f12830b3c17a5a9e4afcc370e3a37e_400x400.jpeg"](https://pbs.twimg.com/profile_images/3647943215/d7f12830b3c17a5a9e4afcc370e3a37e_400x400.jpeg%22) style="min-width: 0px; width: 40px; max-height: 100%; border-radius: 50%; background-position: 50% 50%; background-repeat: no-repeat;">

**How to fix:** 
Fix any of the following:
  Element does not have an alt attribute
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation

**User Impact:**
Users with AT will face difficulty if select element does not have an alt attribute.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

![WCAG_1 1 1-Ensures img elements have alternate text or a role of none or presentation](https://github.com/microsoft/AdaptiveCards/assets/95913748/2e75dddb-f12b-4437-a458-c32d47418d50)

![WCAG1 1 1-Ensures img elements have alternate text or a role of none or presentation](https://github.com/microsoft/AdaptiveCards/assets/95913748/d268c52a-45fa-4445-b4f2-393a9d5b9709)

![Latest](https://github.com/microsoft/AdaptiveCards/assets/95913748/7b086e81-249a-4269-bf3d-e0d1f7f1906f)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Hi @anna-dingler,

As this issue is still repro, so could you please let us know is there any future plan to fix this issue?

Snippet:

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/6b4840b0-dd82-4e3d-866c-15a851b5b28a)

**Comment by anna-dingler** [comment]
Hi @vagpt,

This issue is fixed on our internal site. I expect us to make another website release this month, so I am marking this as completed.

Please let me know if you have any questions!

**Comment by vagpt** [comment]
This issue is not repro hence adding A11yTTValidated tag.

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/caaaadda-54fd-4fd8-8ad2-97737d091129)
